qiskit/test/mock/backends/poughkeepsie/fake_poughkeepsie.py

```diff
@@ -70,3 +56,14 @@ def properties(self):
         with open(os.path.join(dirname, filename), "r") as f_prop:
             props = json.load(f_prop)
         return BackendProperties.from_dict(props)
+
+    def defaults(self):
+        """Returns a snapshot of device defaults as recorded on 11/15/19.
```

Comment:
These lines were manually added, so the date isn't correct anymore. I think they should just be removed from all the `.py` files - the JSON fields each have the recorded time of the property anyway.

I just thought a fake backend user might find this info useful. Not sure if your script can edit docstring for each `FakeBackend` class to say this - it's non-blocking.

Reply:
The script isn't that smart. I did the bare minimum to get it working, it just loops over the backends return from the ibmq provider and regenerates the json files for each of them (or those that are specified). We can  get more involved with it, and have it generate the fake_*.py files too they're mostly static anyway so it would be pretty easy

Reply:
it's ok i don't think it's worth the effort but I do think those hardcoded dates should be removed.